### PR TITLE
Log added/updated/deleted sections only if they are not empty

### LINKED ISSUE #4: Objects are logged as "Added" and "Deleted", even if empty
Whenever I update the state, without adding or removing anything, those sections are still logged to the console:

screen shot 2018-09-16 at 03 51 56

I see you're checking if there are no additions or deletions:

https://github.com/sindresorhus/unstated-debug/blob/3d6f4b5f1e67b2b71b71f48f3a63c308c4020042/index.js#L44

https://github.com/sindresorhus/unstated-debug/blob/3d6f4b5f1e67b2b71b71f48f3a63c308c4020042/index.js#L52

But these lines don't take into account empty objects.

Would you consider a PR adding support for detecting empty objects?

### PR DESCRIPTION
**Pull request by fabiospampinato**
Closes #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -43,15 +44,15 @@ __SUPER_SECRET_CONTAINER_DEBUG_HOOK__(container => {
 		const group = UNSTATED.isCollapsed ? console.groupCollapsed : console.group;
 		group(name);
 
-		if (diff.added) {
+		if (!isEmpty(diff.added)) {
```
Comment (sindresorhus) [inline index.js]: You don't really need Lodash for this. Can just use `if (Object.keys(diff.added).length > 0) {`

**Comment by sindresorhus** [comment]
Can you give the PR a proper descriptive title?

**Comment by fabiospampinato** [comment]
Updated PR title and avoiding using lodash.

Let me know if you want any other changes to be made 👍

**Comment by sindresorhus** [comment]
👌

**Pull request merged**